internal/xds/translator/cluster.go

```diff
@@ -83,7 +84,7 @@ func buildEndpointType(settings []*ir.DestinationSetting) EndpointType {
 func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
 	cluster := &clusterv3.Cluster{
 		Name:            args.name,
-		DnsLookupFamily: clusterv3.Cluster_V4_ONLY,
+		DnsLookupFamily: getDNSLookupFamily(args.settings),
```

Comment:
Should we use "clusterv3.Cluster_ALL"?

Reply:
Can we adopt an idea like this? This is the dns selection method of isio.
![image](https://github.com/user-attachments/assets/04b04f48-9f51-4244-a533-0067e6ac9cea)
![image](https://github.com/user-attachments/assets/d801e907-e845-42df-a4d7-d164311faebe)

Reply:
@juwon8891 We can use the `EnvoyProxy.IPFamily` to determine the `DnsLookupFamily`.

Reply:
@zhaohuabing `EnvoyProxy.IPFamily` is meant to configure Envoy listener and Envoy Proxy service config, not backend endpoint config, which is determined based on info in the backendRef `Service` and its corresponding `EndpointSlice` resources

Reply:
the API can be made clearer to avoid this confusion

Reply:
Oh, I see. I got the EnvoyProxy listener IP family settings with the Backend IP families.